client/src/Components/CommonComponent/Table/Table.jsx

```diff
@@ -71,7 +70,9 @@ class TableCmponent extends Component {
             dataSource={columns}
             pagination={{
               pageSize: isNaN(this.state.pageSize)
-                ? columns.length
+                ? columns
+                  ? columns.length
```

Comment:
@Anies12 This is a little hard to follow, when is it supposed to use `columns.length`?

Reply:
if the pageSize is a number and there exist columns

Reply:
maybe `isNaN(this.state.pageSize) && columns ? columns.length : parseInt(this.state.pageSize)` would work?

Reply:
yeah, it would :) , this is a lot easier to read

Reply:
Cool.  Whenever I see or write a nested ternary like that, I try to find a way to refactor it.